### ISSUE TITLE
Version 1.2.1 does not pause the game (in my testing).

### ISSUE THREAD

**Issue by bo-sendi** [cabally/CommandPause#2]
Title.

Version 1.2.1 (for improved BetterUI compatibility) does not appear to work properly, on a clean setup without changing any config. No matter what I do, the game does not pause while a Command selection window is open. I have also tried tweaking the config for BetterUI (since it is required as a dependency for this mod) and even disabling its Command component entirely, and I still cannot get the game to pause.

Any advice or solutions would be greatly appreciated, since I cannot get this mod to work under any circumstance (possibly a settings issue?). Also, I'm not sure why ItemStats and BetterUI are listed as required dependencies through the mod manager, while the description says they are recommended. Consider removing the required dependency if they are not necessary :)

Version 1.2.0 also does not seem to work for me, completely disabling BetterUI (and only using ItemStats + this mod).

I tested a few times using Mul-T and only the command artifact, and could not get this mod to work at all unfortunately. Also tried a single run with Engineer and also couldn't get it to work. Was really looking forward to it.

**Comment by pierre-cantegril** [comment]
Hi,
I ran into the same problem. 1.2.0 and 1.2.1 don't work for me. Using the modmanager (r2modman), I uninstalled the mod and installed the 1.1.0 version and this one works with the Anniversary Update

**Comment by Bogdanov89** [comment]
Yea the latest version does not work for me either, just using betterUI.

**Comment by cblly** [comment]
hey sorry about the late reply I'v been busy with college. Do you know if this occurs with version 2.0?

**Comment by bo-sendi** [comment]
No worries. Just tested it using a mod profile with CommandPause as the only mod (with dependencies) and it does not appear to pause whatsoever. The invincibility appears to work, as I do not take damage with the command menu open, but enemies will still continue to spawn and attempt to attack me; so I don't think the invisibility part works, at least as mentioned on the mod page.

It most certainly does not pause, though; and I have not seen a config file generated after running the mod, so I can't test with alternative settings either. Essentially, the mod works for its main function (freely spend time choosing Command items), but doesn't work perfectly as enemies will still continue to spawn and attempt to attack you, as well as the timer/difficulty increase still ticking. Waiting on an item for too long (especially in early stages or with few items), it's actually very possible to get one-shot by a group of enemies just standing next to you, waiting for you to no longer be invincible 😢

TL;DR - The mod does *not* properly pause (nor apply invisibility) while in a Command menu, but it does properly apply invincibility, giving you the opportunity to choose freely (at the cost of time and potentially a lot of threats surrounding you as you leave).

**Comment by Bogdanov89** [comment]
oh i did not notice that.
 well that is still very useful, thank you!

**Comment by cblly** [comment]
do you think I should leave it as is?

I'v talked with multiple people and balancing because it's a bit weird to balance. Should I either A) make it so that it give invicibility so it works in multiplayer but it costs time to use(invisibility is supposed to also be part of it but I never got it working.)  or B) make it a single player thing so the time is paused.

**Comment by Bogdanov89** [comment]
personally i dont care about time so the first option that also works in multiplayer is imho better.
